apps/wing-console/console/ui/src/ui/blue-screen-of-death.tsx

```diff
@@ -1,44 +1,41 @@
 import classNames from "classnames";
-import { useEffect, useRef } from "react";
+import { useEffect, useRef, useState } from "react";
 
 import { formatAbsolutePaths } from "../features/console-logs.js";
 
 export const BlueScreenOfDeath = ({
   title,
   error = "",
-  hidden = false,
   displayLinks = true,
   displayWingTitle = true,
 }: {
   title: string;
   error: string;
-  hidden?: boolean;
   displayLinks?: boolean;
   displayWingTitle?: boolean;
 }) => {
-  const errorRef = useRef<HTMLElement>(null);
+  const [formattedPathsError, setFormttedPathsError] = useState<string>("");
```

Comment:
Typo in `setFormttedPathsError`. Also, no need to specify `<string>` since it's deduced by the default `""` you pass to the function.